backend/app/services/rag/splitter/validators.py

```
@@ -0,0 +1,112 @@
+        result: ValidationResult from validate_markdown_chunks
+
+    Returns:
+        Error response dict with error_code, error_message and details
+    """
+    # Find the largest chunk for the error message
+    max_chunk = max(result.oversized_chunks, key=lambda x: x.token_count)
+
+    return {
+        "error_code": "CHUNK_TOO_LONG",
+        "error_message": (
+            f"检测到 Markdown 中某些章节内容过长（约 {max_chunk.token_count} tokens）。"
```

Ruff (0.14.13)

String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)